src/assets/translations/en/main.json

```diff
@@ -2800,7 +2800,7 @@
     },
     "foxWifHat": {
       "title": "FOX Wif Hat",
-      "description": "It's just FOX wif a hat, don't overthink it. Enjoy a limited time on-chain discount program. As your discount decays learn about FOX on the rest of the page and swap into FOX any time to preserve your fee discount. Whichever holding is higher saves you money on the ShapeShift platform.",
+      "description": "Claim your FOXwifhat. Act fast - discounts decay over time. The more FOX or FOXwifhat you hold, the more you save. Stack more FOX tokens for continuous savings plus revenue sharing, governance, and yield farming perks. Learn more about those programs on the rest of this page..",
```

Comment:
```suggestion
      "description": "Claim your FOXwifhat. Act fast - discounts decay over time. The more FOX or FOXwifhat you hold, the more you save. Stack more FOX tokens for continuous savings plus revenue sharing, governance, and yield farming perks. Learn more about those programs on the rest of this page.",
```

Reply:
thank you! thought i committed that but i guess i forgot to save a second time.

Reply:
Don't worry 😄